bitcoin/src/blockdata/transaction.rs

```diff
@@ -846,14 +851,15 @@ impl Transaction {
         Ok(())
     }
 
-    /// Is this a coin base transaction?
+    /// Checks if this is a coin base transaction.
     pub fn is_coin_base(&self) -> bool {
         self.input.len() == 1 && self.input[0].previous_output.is_null()
     }
 
-    /// Returns `true` if the transaction itself opted in to be BIP-125-replaceable (RBF). This
-    /// **does not** cover the case where a transaction becomes replaceable due to ancestors being
-    /// RBF.
+    /// Returns `true` if the transaction itself opted in to be BIP-125-replaceable (RBF).
+    ///
+    /// This **does not** cover the case where a transaction becomes replaceable due to ancestors
+    /// being RBF.
```

Comment:
Perhaps we should add a note that all transactions [may be replaceable anyway](https://bitcoinops.org/en/newsletters/2022/10/19/#transaction-replacement-option). The function name is already good.

Reply:
Done, please check what I wrote.